config/features.yml

```
@@ -286,7 +286,11 @@ features:
   show_chapter_36:
     actor_type: user
     description: >
-      This will toggle between an eBenefits link and a VA.gov link for the VR&E chapter 36 form
+      This will toggle between an eBenefits link and a VA.gov link for the VR&E chapter 36 for
```

```suggestion
      This will toggle between an eBenefits link and a VA.gov link for the VR&E Chapter 36 form
```

I think you dropped an `m` ?